src/Microsoft.FeatureManagement/FeatureFilters/PercentageFilter.cs

```diff
@@ -21,9 +21,9 @@ public class PercentageFilter : IFeatureFilter, IFilterParametersBinder
         /// Creates a percentage based feature filter.
         /// </summary>
         /// <param name="loggerFactory">A logger factory for creating loggers.</param>
-        public PercentageFilter(ILoggerFactory loggerFactory)
+        public PercentageFilter(ILoggerFactory loggerFactory = null)
         {
-            _logger = loggerFactory.CreateLogger<PercentageFilter>();
+            _logger = loggerFactory?.CreateLogger<PercentageFilter>();
```

Comment:
Prior to this PR, before we have a default `null` value for loggerFactory, am I allowed to explicitly pass a null as parameter, i.e. `new PercentageFilter(null)`? If so, this PR also fixes the potential NPE, right?

Reply:
Yes.